sdk/schemaregistry/schema-registry-avro/test/public/utils/mockedRegistryClient.ts

```diff
@@ -137,3 +143,31 @@ export function createTestRegistry(
       })
     : createMockedTestRegistry();
 }
+
+export async function removeSchema(
+  schemaName: string = testSchemaName,
+  groupName: string = testGroup,
+  apiVersion: string = "2022-10"
+): Promise<void> {
+  if (!isLiveMode()) {
+    return;
+  }
+
+  const endpoint = assertEnvironmentVariable("SCHEMAREGISTRY_AVRO_FULLY_QUALIFIED_NAMESPACE");
+  const url = `${endpoint}/$schemagroups/${groupName}/schemas/${schemaName}/?api-version=${apiVersion}`;
+  const DEFAULT_SCOPE = "https://eventhubs.azure.net/.default";
+  const request = createPipelineRequest({
+    url,
+    method: "DELETE",
+    timeout: 0,
+    withCredentials: true,
+    headers: createHttpHeaders({}),
+    allowInsecureConnection: new URL(url).protocol !== "https:",
+  });
+  const client = createDefaultHttpClient();
+  const pipeline = createEmptyPipeline();
+  const credential = createTestCredential();
+  const authPolicy = bearerTokenAuthenticationPolicy({ credential, scopes: DEFAULT_SCOPE });
+  pipeline.addPolicy(authPolicy);
+  await pipeline.sendRequest(client, request);
```

Comment:
I think the pipeline can be cached instead of creating it every time we send the request.